modules/ui/src/main/scala/lucuma/ui/components/GppStyles.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.components
+
+import react.common.Css
+
+trait GppStyles {
```

Comment:
Why the name `GppStyles`? It looks like this is only meant to be used internally by the MoonPhase component? Maybe it could be `MoonStyles`?

Reply:
Seems like a good idea